addons/binding/org.openhab.binding.kodi/ESH-INF/i18n/kodi_de.properties

```diff
@@ -0,0 +1,38 @@
+# binding
+binding.kodi.name = Kodi Binding
+binding.kodi.description = Das Kodi Binding erlaubt das Kodi Media Center zu steuern 
+
+# thing types
+thing-type.kodi.kodi.label = Kodi Media Center
+thing-type.kodi.kodi.description = Kodi Media Center Binding
+thing-type.config.kodi.kodi.ipAddress.label = Netzwerkaddresse
+thing-type.config.kodi.kodi.ipAddress.description = IP Addresse oder Host Name des Kodi Media Centers
+thing-type.config.kodi.kodi.port.label = Port
+thing-type.config.kodi.kodi.port.description = Port des Web Socket Services des Kodi Media Centers
+
+# channel types
+channel-type.kodi.volume.label = Lautst�rke
+channel-type.kodi.volume.description = Lautst�rke des Kodi Media Centers
+channel-type.kodi.mute.label = Stumm schalten
+channel-type.kodi.mute.description = Das Kodi Media Centers stumm schalten
+channel-type.kodi.control.label = Kontrolle
+channel-type.kodi.control.description = Steuert das Kodi Media Center (z.B. Play, Pause, Stop)
+channel-type.kodi.playuri.label = URI abspielen
+channel-type.kodi.playuri.description = Spielt einen URI ab
+channel-type.kodi.shownotification.label = Nachricht anzeigen
+channel-type.kodi.shownotification.description = Zeigt eine Nachricht auf dem Bildschirm an
+channel-type.kodi.input.label = Tastendruck
+channel-type.kodi.input.description = Schickt einen Tastendruck an das Kodi Media Center um auf dem Bildschirm zu navigieren
+channel-type.kodi.inputtext.label = Texteingabe
+channel-type.kodi.inputtext.description = Schickt eine Zeichenkette als Eingabe an das Kodi Media Center
+channel-type.kodi.title.label = Titel
+channel-type.kodi.title.description = Titel des aktuellen St�cks oder Films
+channel-type.kodi.title.label = Titel
+channel-type.kodi.title.description = Titel des aktuellen St�cks oder Films
+channel-type.kodi.album.label = Album
+channel-type.kodi.album.description = Album des aktuellen St�cks
+channel-type.kodi.artist.label = K�nstler
```

Comment:
You have stored the file with text encoding ISO 8859-1. As a result the german "umlauts" do not appear correctly (in Chrome on Windows) neither in Paper UI nor in Basic UI. I have changed the encoding to UTF-8 and corrected the "umlauts". Then everything looked fine.
@kaikreuzer I have tried to find a hint on the correct encoding in the developer documentation for Smarthome and openHAB but without success . Java sources and XML files are all using UTF-8 but all properties files in the openHAB repository that I have checked where using encoding ISO 8859-1.